01week/datatypesGUI.js

```diff
@@ -80,9 +84,9 @@ function isTypeOf(data) {
   return console.log(typeof data);
 }
 
-isTypeOf(bool);
-isTypeOf(nothing);
-isTypeOf(dunno);
-isTypeOf(eighteen);
-isTypeOf(notNum);
-isTypeOf(helloWorld);
+// isTypeOf(bool);
+// isTypeOf(nothing);
+// isTypeOf(dunno);
+// isTypeOf(eighteen);
+// isTypeOf(notNum);
+// isTypeOf(helloWorld);
```

Comment:
I like the way you wrote this, just to go a step further: what instances would you get a datatype back? For instance: var myBool = (10>9), this would return a true which is a boolean.